elevation_mapping/include/elevation_mapping/ElevationMapping.hpp

```diff
@@ -105,6 +105,14 @@ class ElevationMapping
    */
   bool getSubmap(grid_map_msgs::GetGridMap::Request& request, grid_map_msgs::GetGridMap::Response& response);
 
+  /*!
+   * ROS service callback function to return a submap of the elevation map.
```

Comment:
Same comment as getSubmap(), can you add a comment that shows the difference to getSubmap()